docs/custom.md

```diff
@@ -106,6 +107,31 @@ happoCustom.registerExample({
 });
 ```
 
+#### Waiting for content
+
+In some cases, examples might not be ready by the time Happo takes the
+screenshot. Although adding a delay could help, it will only work well if the
+asynchronous event is consistently timed. In these cases the `waitForContent`
+parameter might help.
+
+Let's assume that we have an example that needs to fetch some external data. In
+order to wait for the data to finish loading, we can add a `waitForContent`
+parameter with some unique string from the loaded state:
+
+```js title="main.js"
+happoCustom.registerExample({
+  component: 'Description',
+  variant: 'loaded',
+  render: async () => {
+    document.body.innerHTML = '<div id="description">Loading...</div>';
+    const data = await fetch('http://example.com/api/data');
+    document.querySelector('#description').innerHTML =
+      `<p>Description: ${data.description}</p>`;
```

Comment:
This example is pretty contrived and I'm not sure the waitForContent is even necessary here because the fetch is awaited inside of render. Let me know if you have ideas for a better example here.